### ISSUE TITLE
Manage custom blocking list

### ISSUE THREAD

**Issue by ThatIsAPseudo** [OxfordHCC/tracker-control-android#200]
As far as I understand, I can add local blocking lists to append it after already existing ones. However I cannot see what lists I already have, nor deleting only one among the others.
IMO there should be a way to manage custom blocking list, as well as adding several URLs for remote blocking lists.

**Comment by ChunkyProgrammer** [comment]
So you want to be able to manage lists like how uBlock Origin does it?

![1zrG4C0OKeMOG1UWU9RKFT5EErD8aQHsGx5coN3786Q](https://user-images.githubusercontent.com/78101139/119135786-e0debe00-ba0c-11eb-8310-7da07a1d36a9.png)

**Comment by ThatIsAPseudo** [comment]
Yes that's what I'm thinking of
DNS66 has this feature where you can select which rules you want to activate

**Comment by kasnder** [comment]
Good points. The problem with these lists is that they don't contain information on the companies behind.

**Comment by ThatIsAPseudo** [comment]
Yes that information would be missing, though the main reason why I use TC is that it does pretty much the same ad/tracking blocking as DNS66, but with a connection log in addition.

**Comment by kasnder** [comment]
> Yes that information would be missing, though the main reason why I use TC is that it does pretty much the same ad/tracking blocking as DNS66, but with a connection log in addition.

Oh, super interesting! At the moment, I don't have enough free time to add this feature, but agree that it'd be really important to have.

**Comment by sbstn87** [comment]
I second this one, it would be super useful. The missing company information is not a big issue.

It would be great if TC could manually block/unblock selected connections - Blokada 5 does it, where you can tap on any connection in the log and choose to block or unblock it. Very useful if the block list is too broad and does not allow you to connect to a legit server.

**Comment by PawelGu** [comment]
Oh yes! This is the only function missing keeping it from perfect.
Sometimes there are connections which I want to be blocked but don't need any further information about. Pretty please :)

**Comment by sverley** [comment]
Yes, a block list based on url could be good. Also a white list could be usefull as far as you force to block "Essentials" by default and also for "System service". It is too much to ask to the user to allow/block the same url in all apps. Maybe you could build a list of all url found and propose 3 choices : block/allow/app specified
